src/components/list/list.scss

```diff
@@ -295,8 +295,19 @@ md-list-item {
       align-items: center;
 
       height: 100%;
```

Comment:
Should this be changed to auto? I just had an issue where the height of the secondary container was very large. I fixed it by setting height to auto... by overriding the private style :(